src/Adapter/Product/Update/ProductTagUpdater.php

```diff
@@ -101,7 +102,7 @@ public function setProductTags(Product $product, array $localizedTagsList): void
 
         // Since tags have been modified we need to update the indexation values (only for active products)
         if ($product->active) {
-            $this->productIndexationUpdater->updateIndexation($product);
+            $this->productIndexationUpdater->updateIndexation($product, ShopConstraint::allShops());
```

Comment:
Maybe add a comment here explaining that tags do not have multishop data so we don't have a shopId to build a constraint here and that's why we use an all shops constraint